VMBackup/main/NodeBased/service_metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "Unit": {
+      "Description": "Long running Snapshot service for Microsoft Azure Restore Points",
+      "After": "multi-user.target"
+    },
+    "Service": {
+      "Type": "simple",
+      "Restart": "always",
+      "WorkingDirectory": "../..",
+      "ExecStart": ["/usr/bin/env", "python", "main/NodeBased/PollingService.py"]
```

Comment:
updated to IaaSExtensionSnapshotService. Added the 'Snapshot' instead of 'Linux' as it would be clearly distinguished between all the services registered in /etc/systemd/system/.